### DISCUSSION TITLE
Regarding student account and personal account

### DISCUSSION THREAD

**Discussion by san-jib-dahal** [community/community#57397] [GitHub Education]
### Hi everyone,

**Hello everyone ,I need help with a student developer pack.**
I have my own personal GitHub that I am using since many years and contains many useful repositories. 
A week ago, I got my university email and unfortunately and without thinking, I created GitHub account with my school email but that email will be deleted as soon as I graduate. 
So, can I delete that school account and add it to my personal account as secondary email and get that student developer pack or it will not work?
Please reply to my problem and help me.

**Comment by samus-aran** [comment]
Hi @san-jib-dahal 

Apologies for the delayed response from the team. If you haven't already applied using your new GitHub account with the academic email I would suggest you add that academic email to your main GitHub account.
If you have already used the academic email on your new GitHub account and received a successful Student verified it’s not possible to transfer Global Campus benefits over from one GitHub account to another.

However, you are able to merge your accounts, but the account with the benefits would be the main account. You can add your non-academic email from the previous account to the account containing the academic email. If you desire you can then rename this account once merged. For more details check out out the [merging of your personal accounts](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-your-personal-account/merging-multiple-personal-accounts) documentation.

**Reply by juliogarciape** [reply]
> **A Query** I have decided to delete my GitHub account which has a GitHub Student Developer pack, if I create a new GitHub account and apply for GitHub Student Developer pack will I get acceess to it? Please respond to my query as soon as possible and help me out.

Hi, did you manage to delete your account and test if the benefits are obtained again from your main account? I'm thinking of doing the same but I don't know what result I'll get.
